modules/service/node-service.js

```diff
@@ -0,0 +1,13 @@
+class NodeService {
+    constructor(ctx) {
+        this.config = ctx.config;
+        this.logger = ctx.logger;
+    }
+
+    stop() {
```

Comment:
Add variable exitCode to stop method, since we will call this method from different places in code with different exit codes.

Reply:
Fixed with [this commit](https://github.com/OriginTrail/ot-node/pull/1699/commits/85768db8962b171f92711922d0946e7a82585d28)